.github/workflows/codeql-analysis.yml

```
@@ -70,3 +70,60 @@ jobs:
 
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v3
+            - name: Setup Node.js environment
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)